modes.xml

```diff
@@ -82,7 +82,7 @@
   
   <mode name="es-pt" install="yes">
     <pipeline>
-      <program name="lt-proc">
+      <program name="lt-proc -w">
         <file name="es-pt.automorf.bin"/>
       </program>
       <program name="apertium-extract-caps"/>
```

Comment:
https://wiki.apertium.org/wiki/Capitalization_restoration#Extracting_Capitalization

extract should be after tagger